src/python/pants/option/global_options.py

```diff
@@ -1602,6 +1608,35 @@ class BootstrapOptions:
             """
         ),
     )
+    remote_client_certs_path = StrOption(
```

Comment:
Any reason for not including "tls" in the name similar to how Bazel names it? I.e., `remote_tls_client_certs_path` and `remote_tls_client_key_path`? I guess the choice was made with `remote_ca_certs_path` not including `tls` in the name.

Reply:
Yeah; I think all fields should have `tls` or none, yes.